nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,97 @@
+const { request, response } = require("express");
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/:id", async (request, response) => {
+  const mealId = parseInt(request.params.id);
+
+  if (mealId <= meals.length) {
+    const findMealWithId = meals.find(meal => meal.id == mealId)
+    response.send(findMealWithId);
+  }
+  else if (isNaN(mealId)) {
+    response.status(400).send(`Sorry. The meal id you requested should be an integer`)
+
+  }
+  else {
+    response.send();
```

Comment:
I think you can add a response like this here: `response.status(404).send(`Sorry. The meal id you requested doesn't exist`)`

Reply:
I agree with you Florin and I did the same before. But the test demands us to return nothing, so I did this.